website/docs/admin_settings.md

```diff
@@ -7,12 +7,15 @@ sidebar_label: Working with settings
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-OpenPype stores all of it's settings and configuration in the mongo database. To make the configuration as easy as possible we provide a robust GUI where you can access and change everything that is configurable
+OpenPype stores all of its settings and configuration in the mongo database. To make the configuration as easy as possible we provide a robust GUI where you can access and change everything that is configurable
 
 **Settings** GUI can be started from the tray menu *Admin -> Studio Settings*.
 
-Please keep in mind that these settings are set-up for the full studio and not per-individual. If you're looking for individual artist settings, you can head to
-[Local Settings](admin_settings_local.md) section in the artist documentation.
+:::important Studio Settings versus Local Settings
+Please keep in mind that these settings are set up for the full studio and not per-individual. If you're looking for individual artist settings, you can head to
+[Local Settings](admin_settings_local.md) section in the documentation.
```

Comment:
This leads up that looks like an orphan page for the local settings. Maybe have that part of `artist_install` or after that?

Reply:
So it points to [this](https://openpype.io/docs/admin_settings_local) and yes, it's not pretty. However, I'd actually say that the issue then becomes that the Local Settings are just not correctly documented yet. We should start to do that but can be separate from these small fixes. What do you think?

Reply:
Cool, if you could move it, I'll eventually get to expanding on the Local Settings from this issue; https://github.com/ynput/OpenPype/issues/4620

Reply:
> Cool, if you could move it

Move it how? :)

Reply:
Sorry, think the best place might here:

![Capture](https://user-images.githubusercontent.com/1860085/224985825-1cb2d9f8-f255-431e-9f74-0a8af7f64f24.PNG)

Reply:
Yeah, fair. Can move it with the PR.